INSTALL_MACOS.md

```diff
@@ -449,10 +463,10 @@ qmake -r CONFIG+=debug
 
 It is possible to build Natron using clang (version 3.8 is required,
 version 9.0 is recommended) with OpenMP support on
-MacPorts (or Homebrew for OS X 10.9 or later).  OpenMP brings speed improvements in the
+MacPorts (or homebrew for MacOS 10.9 or later).  OpenMP brings speed improvements in the
```

Comment:
Why is Homebrew no longer capitalized?

Reply:
Accidental, I was trying to learn how to use Emacs while doing these commits, and that caused a lot of missed issues in terms of formatting